src/migrations/20191114103355-change-utxo-table-index.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.removeIndex("UTXOs", "u_t_x_os_address_id");
+        await queryInterface.removeIndex(
+            "UTXOs",
+            "UTXOs_assetType_usedBlockNumber"
+        );
+
+        await queryInterface.addIndex("UTXOs", [
+            "address",
+            "blockNumber",
+            "transactionIndex",
+            "transactionOutputIndex"
+        ]);
+        await queryInterface.addIndex("UTXOs", [
+            "assetType",
+            "blockNumber",
+            "transactionIndex",
+            "transactionOutputIndex"
+        ]);
```

Comment:
I think adding the `usedBlockNumber` column here helps the performance of the snapshot API.

Reply:
It is hard to paginate if we use `usedBlockNumber` as index.

Reply:
I see. Let's leave it as is.